NMS-15051: Add KPI for appliance count by model (#5567)
OpenNMS/opennms: docs/modules/operation/pages/deep-dive/admin/housekeeping/introduction.adoc

It publishes them on system startup and every 24 hours after.
Usage statistics include the following data points:

* System ID (a randomly generated, universally unique identifier (UUID)).
* {page-component-title} release.
* {page-component-title} version.
* OS architecture.
* OS name.
* OS version.
* Number of available processors (CPU).
* Amount of free physical memory.

> * Amount of total physical memory.

Reviewer: ```suggestion
* Total amount of physical memory.
```

> * Number of alarms in the `alarms` table.

Reviewer 2: ```suggestion
* Number of alarms in the alarms table.
```

> * Number of situations in the `alarms` table.

Reviewer 2: ```suggestion
* Number of situations in the alarms table.
```

> * Number of events in the `events` table.

Reviewer 2: ```suggestion
* Number of events in the events table.
```

> * Number of IP interfaces in the `ipinterface` table.

Reviewer 2: ```suggestion
* Number of IP interfaces in the ipinterface table.
```

> * Number of nodes in the `node` table.

Reviewer 2: ```suggestion
* Number of nodes in the node table.
```

> * Number of nodes, grouped by System OID.

Reviewer 2: ```suggestion
* Number of nodes, grouped by system OID.
```

* Number of locations.
* Number of Minions.

> * Number of provisiond requisitions by scheme.

Reviewer: ```suggestion
* Number of provisiond requisitions, by scheme.
```

* Provisiond thread pool sizes.

> * List of enabled/disabled services.

Reviewer: ```suggestion
* List of enabled and disabled services.
```

* List of installed features in Karaf.
* Global notification enablement status.

> * Destination path count.
> * On-call role count.

Reviewer: ```suggestion
* Number of destination paths.
* Number of on-call roles.
```

* Relational Database Management System (RDBMS) name.
* RDBMS version.

> * Pollerd tasks completed.
> * Event logs processed count.

Reviewer: ```suggestion
* Number of pollerd tasks completed.
* Number of event logs processed.
```

> * Core Flows persisted count
> * Core Newts samples inserted count
> * Core Queued updates completed

Reviewer: In line with this change and the one immediately below, all entries in this list should have a period at the end of the sentence.

```suggestion
* Number of core flows persisted.
* Number of Newts samples inserted.
* Number of queued updates completed.
```

> * Number of Virtual Appliance, Appliance Mini, and Appliance 1U connected to the system

Reviewer: ```suggestion
* Number of virtual appliances, Mini appliances, and 1U appliances that are connected to the system.
```
